src/main/java/tests/FruitTransactionTest.java

```diff
@@ -0,0 +1,21 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
Avoid using wildcard imports. It's better to import only the classes that are needed. This will improve readability and understanding of which classes are actually being used in this test.